name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/SecuritiesPerformanceView.java

```diff
@@ -886,6 +886,22 @@ public String getText(Object r)
         });
         column.setSorter(ColumnViewerSorter.create(SecurityPerformanceRecord.class, "periodicitySort")); //$NON-NLS-1$
         recordColumns.addColumn(column);
+
+        // dividend yield on cost in der wertpapier performance detailansicht
```

Comment:
Comments should be relevant – I don’t think this one adds anything. In any case, they should be in English.

Reply:
I agree with your comment comments being in English. But I also have to admit that I haven't been too strict with this policy in the past - and this class actually contains a number of German comments that slipped in through contributions. No worries - let's remove the Germany part of the comment

Reply:
Yes, this was more a comment for myself to remember what part of UI I'm changing here. I removed it.